packages/beacon-node/src/chain/options.ts

```diff
@@ -94,6 +95,7 @@ export type SeenCacheOpts = {
 export const defaultChainOptions: IChainOptions = {
   blsVerifyAllMainThread: false,
   blsVerifyAllMultiThread: false,
+  blsAddVerificationRandomness: true,
```

Comment:
as a convention we want to have flags as disabled by default similar to `disableBlsBatchVerify`, so its like `disable*`
also this is for same message verification only so should be `*sameMessage*`

Reply:
Updated to `disableSameMessageVerificationRandomness`